ext/com_dotnet/com_com.c

```diff
@@ -39,7 +39,7 @@ PHP_METHOD(com, __construct)
 	CLSID clsid;
 	CLSCTX ctx = CLSCTX_SERVER;
 	HRESULT res = E_FAIL;
-	int mode = COMG(autoreg_case_sensitive) ? CONST_CS : 0;
+	int mode = 0;
```

Comment:
I wonder if we should just inline that `0`.

Reply:
Yes, I changed this. We can also simplify function `php_com_import_typelib` with the removal of `CONST_CS`, but I'm now learning how PHP source code works and don't want to rush.

Reply:
Thank you! I don't think that we can simplify `php_com_import_typelib`, since the `mode` is checked there for `CONST_PERSISTENT` which is still supported.